templates/app/aspnet-core/src/MyCompanyName.MyProjectName.DbMigrator/appsettings.json

```diff
@@ -1,6 +1,6 @@
 {
   "ConnectionStrings": {
-    "Default": "Server=(LocalDb)\\MSSQLLocalDB;Database=MyProjectName;Trusted_Connection=True;TrustServerCertificate=True"
+    "Default": "Server=özer\\OZER;Database=MyProjectName;Trusted_Connection=True;TrustServerCertificate=True"
```

Comment:
Should you revert that? we don't want to change connection string with your server name :)